### ISSUE TITLE
Battle spells problems (bugs, weakness, usability)

### ISSUE THREAD

**Issue by uhloin** [Sirttas/ElementalCraft#115]
Version elementalcraft-1.19.2-5.4.6.jar

I admit that maybe I didn’t guess about something, then I apologize. But current version of battle spells is not better than if I just hit mobs by sword or shoot with bow. What a reward for search, collect and spend mana?

**All battle spells*
They have 2 flaws agains normal weapon: big cooldown and cost of mana. To be competitive with regular weapon one of these barriers should be removed or weakened. 

May be I already is spoiled by other magic mod "Ars Nouveau". I do not say it is better than yours. mods are too different to compare and the diversity is good. I just want make example of limitation mechanism to avoid overuse of magic. AN has auto-recover mana mechanics and zero cooldown for spells. The mana limit is enough to avoid overuse of magic. After several quick "hits" mana becomes zero and battlemage is forced to make pause. In your mod mana is not recovered and there are cooldowns. I think if you do not want copy and keep your own way then yours spells should save huge cooldowns but make proportionally huge damage. 1 hit should be enough for zombie and skeletons. And capacity of mana should be enough for at least one night of fight. 

Just as a dream = may make some "charge" mechanics: longer preparation with hold button = bigger effect *ba-da-bum* :)

**Translocation** = does not work
Checked in superflat world, creative mode. Both warstaff and focus, left and right click, target mob and grass => nothing happens.
Update: it works in usual survive mode with use scroll itself. But target of teleportation is random in selected direction, not a place pointed by crosshair.

**Enderstrike** = works partially
Teleport is usable but page also says about "increased damage"... really only 1 hp.

**Fireball** = funny but unusable
It has too small damage for such glorios name as "fireball" :) Should be like a creeper explosion with the same cooldown and mana cost.

**Inferno** = unusable
Other glorious name but absolutely ridiculous damage.

**Fire cleave** = usable only with Jewel of Bear
Without jevel damage is 1 hp. Sword or warstaff with "sweep edge" enchantment make greater damage to group and does not spend "mana" and has smaller cooldown. What sense to do right click (spell) and wait cooldown instead of do faster and stronger regular hits?

**Feather spikes** = usable but buggy
Good damage but it has bugs. I can enchant warstaff with this spell twice and 3 times but without any change of work. Also Jewel of Bear does not increase damage.

**Comment by Sirttas** [comment]
Thanks for this in depth feedback. I will offer a complete answer later, I need to investigate each point one by one. For now note that melee spells strike with your main hand weapon.

**Comment by uhloin** [comment]
"melee spells strike with your main hand weapon"

It is useful info! Damage is really increased if I use normal weapon in main hand and spell focus instead of shield and make right click instead of regular melee hit.

But value of damage is still lesser than damage from weapon. I take diamond axe (9 attack damage) but result of direct hit (not splash) by fire ball is only -5 hp (zombie), -6 hp (wandering trader), -7 hp (his lama). Next I used modded sword with 16 damage, result is again -6 hp (wandering trader).

**Comment by Sirttas** [comment]
> But value of damage is still lesser than damage from weapon. I take diamond axe (9 attack damage) but result of direct hit (not splash) by fire ball is only -5 hp (zombie), -6 hp (wandering trader), -7 hp (his lama). Next I used modded sword with 16 damage, result is again -6 hp (wandering trader).

The main hand damage is only used by ender strike and flame cleave if I remember correctly.

Translocation: I have played a lot with it on forgecraft and it's working correctly, it has a check so you don't get tp underground this may break on superflat. It is also (since 1.19.2) ment to work with translation anchors (not yet documented).

In general I am planning on improving spells,but I need to figure what I want to do and time to do it, a lot of spells are currently outdated and need rework. 

Ender strike: try it with a staff with sharp 5 in main hand, it should OS most simple mobs.

Same with flame cleave. 

Fireball: basically a gahst fireball. It needs a buff. 

Inferno: also needs a buff. 

Feather strike: I like the current state it's at, I may change it so the feathers goes straight instead of falling down. 

TLDR: speells needs love and I know it, I just don't fully know what I want to do with them moving forward.

**Comment by uhloin** [comment]
One more magic mod to compare I playing currently - Iron's Spells and Spellbooks. ISS is not balanced, but at least has config to set cooldown, mana cost and damage value, so it may be balanced manually with progression in specific modpack . May be this is possible solution for your mod too.
